src/view/selection.js

```diff
@@ -117,8 +117,8 @@ export default class Selection {
 	 * @type {engine.view.EditableElement|null}
 	 */
 	get editableElement() {
-		if ( this.rangeCount ) {
-			return this.getFirstPosition().editableElement;
+		if ( this.anchor ) {
+			return this.anchor.editableElement;
```

Comment:
To be honest, I am not sure how `Selection` will work if there are ranges from different view tree roots. `getFirstPosition()` returns first position "place-wise", that means a position which `isBefore()` any other position. It does not to be last focused editable, etc.

I think it's better and more predictible to get `editableElement` from `anchor`. Even more, since I changed `Renderer` to use `anchor` and `focus` during conversion, rather than all ranges.